src/Core/Algorithm/Subdivision/SubdividerUtils.hpp

```diff
@@ -4,6 +4,105 @@
 namespace Ra {
 namespace Core {
 
+// Deal with normals
+
+/// Copy normal \p input property of mesh.
+inline void addPropCopy( const OpenMesh::HPropHandleT<TopologicalMesh::Normal>& input,
+                         TopologicalMesh& mesh,
+                         OpenMesh::HPropHandleT<TopologicalMesh::Normal>& copy ) {
+    // add the new property
+    mesh.add_property( copy, mesh.property( input ).name() + "_subdiv_copy" );
+    // copy values
+#pragma omp parallel for
+    for ( int i = 0; i < mesh.n_halfedges(); ++i )
+    {
+        auto heh = mesh.halfedge_handle( i );
+        mesh.property( copy, heh ) = mesh.property( input, heh );
+    }
+}
+
+/// Create a new property for \p input property of \p mesh on faces.
+inline void addPropCopyF( const OpenMesh::HPropHandleT<TopologicalMesh::Normal>& input,
```

Comment:
The method name is not really reflecting what is done in it. 
Something like `createProp` would be more meaningful.
Is the current name motivated by any naming convention ?